Rules/WAR/Scripts/WAR.as

```diff
@@ -724,6 +724,16 @@ shared class WarCore : RulesCore
 	{
 		// can't lose/modify state if the match is not running
 		if (!rules.isMatchRunning()) { return; }
+		if(rules.get_bool("tutorial"))
+		{
+			CBlob@[] halls;
+			getBlobsByName("hall", @halls);
+			if(halls[1].getTeamNum()!=0&&halls[1].getTeamNum()!=1) 
```

Comment:
This is misleading because the `tutorial` property does not imply that we are currently in a **TTH** tutorial, and this relies on having exactly 2 halls in the whole map.